File: Labs/Lab1(Python to Cpp)/Exercises01_A_solutions/input.cpp
#include <iostream>

using namespace std;

int main(){
    char c;
    while(c != 'q'){
        cout << "Enter a character: ";
        cin >> c;
        if(c == 'a'){
            cout << "You have entered the first letter of the alphabet." << endl;
        }
        else if(c == 'o'){
            cout << "\'o\' could refer to \"object\" or maybe \"output\"." << endl;
        }
        else if(c == 'c'){
            cout << "CONTROL CHARACTER!" << endl;
        }
        else if(c != 'q'){
            cout << "This is an unknown input: " << c << "\nTry \'a\', \'c\', \'o\' or \'q\' to quit." << endl;
        }
    }
}

File: Labs/Lab1(Python to Cpp)/Exercises01_A_solutions/input.py
if __name__ == "__main__":
    c = 's'
    while c != 'q':
        c = input('Enter a character: ')
        if c == 'a':
            print('You have entered the first letter of the alphabet.')
        elif c == 'o':
            print('\'o\' could refer to \"object\" or maybe \"output\".')
        elif c == 'c':
            print('CONTROL CHARACTER!')
        elif c != 'q':
            print('This is an unknown input: " << c << "\nTry \'a\', \'c\', \'o\' or \'q\' to quit.')